### PR TITLE
Style fixes in tf2_geometry_msgs.

### PR DESCRIPTION
**Pull request by clalancette**
Signed-off-by: Chris Lalancette <clalancette@openrobotics.org>

### CODE REVIEW

**Comment by clalancette** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/15383/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/10090/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/13067/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/15591/)

**Comment by ahcorde** [comment]
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/13071/)

**Review by ahcorde — APPROVED** [review]

File: tf2_geometry_msgs/CMakeLists.txt [hunk 1]
```diff
@@ -28,8 +28,20 @@ ament_auto_find_build_dependencies(REQUIRED ${required_dependencies})
...
+  find_package(ament_lint_auto REQUIRED)
+
+  list(APPEND AMENT_LINT_AUTO_EXCLUDE
+    ament_cmake_cppcheck
+    ament_cmake_uncrustify
+  )
+  ament_lint_auto_find_test_dependencies()
+
+  ament_cppcheck(LANGUAGE "c++")
+  ament_uncrustify(LANGUAGE "C++")
```
Comment (aprotyas) [inline tf2_geometry_msgs/CMakeLists.txt]: Just curious, why is this needed? As in, the exclusion and then the separate function call.

**Review by aprotyas — APPROVED** [review]

File: tf2_geometry_msgs/CMakeLists.txt (hunk 1 above)
Comment (clalancette) [reply tf2_geometry_msgs/CMakeLists.txt]: That's a good question.

For reasons I don't understand, when you run `cppcheck` against this package, it thinks that all of the header files are C (not C++), despite them being named `.hpp`.  This causes the checking to fail since it is not valid C code.  To force `cppcheck` to consider them as C++, we exclude it from being found during `ament_lint_auto_find_test_dependencies`, and then configure it manually for C++.

The reason that `uncrustify` is here is because this is the pattern I copied from elsewhere.  But it turns out that `uncrustify` doesn't have a problem detecting the file types, so I'm going to remove that bit.

**Comment by clalancette** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/15387/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/10094/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/13074/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/15599/)

**Comment by clalancette** [comment]
I'm going to accept the two approvals, even though I made a minor change after them.  Since CI is green, merging.  Thanks for the reviews.
